### ISSUE TITLE
Histogram snapshot not reporting values correctly

### ISSUE THREAD

**Issue by rwfaulkner** [rstiller/inspector-metrics#3]
Using the histogram snapshot, I found that sometimes the max values were less then the mean and medium and did not appear to be the true max of values. Further investigation showed the the snapshot.getValues() was not return an array that was properly sorted by numeric order. Further investigation is showing that the call to sort in SimpleSnapshot (https://github.com/rstiller/inspector-metrics/blob/master/lib/metrics/snapshot.ts#L27) is using the default compareFn (lib.es6.d.ts). From there I found that the default is "he elements are sorted in ascending, ASCII character order". You can see this below in the sample values returned. Looks like a proper numeric compare function must be provided to sort.
```
 histogram values: [0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,1,1005,1009,1017,1029,1037,1040,1064,1090,1096,1106,1115,1127,1134,1135,1144,1153,1162,1166,1168,1180,1193,1201,1201,1220,1223,1238,1263,1267,1333,1348,1351,1370,1382,1395,1408,1424,1436,1456,1476,1487,1496,1503,1538,1544,1546,1549,1560,1561,1579,1592,1602,1603,1624,1636,1639,1649,1662,1662,1795,1817,1822,1845,1848,1849,1887,1917,1927,1939,1959,2,2,2,2,2014,2023,2049,2078,2079,2113,2149,2173,2224,2481,2609,2784,3258,3372,3829,4,736,739,749,756,757,781,782,785,785,800,800,806,819,830,836,860,863,874,885,885,906,907,924,949,958,978,986,988,994]
```

**Comment by rstiller** [comment]
Thank's for pointing out the issue and already providing a solution.
I published a new version with the fix: v1.8.6

Please try out and close the ticket if it works for you - if not let me know.

**Comment by rwfaulkner** [comment]
Thanks for the speedy response. Verified that the fix is indeed working in v1.8.6
